### ISSUE TITLE
Just a quick question

### ISSUE THREAD

**Issue by TheLoneDeveloper** [eplightning/hetzner-k3s-tf-ansible#3]
I just made a high availability cluster with 3 master nodes and 4 workers few days ago.

I'm just wondering how you scale up all the workers without going to hetzner and doing it manually?

Do i just change the worker server type in the terraform/terraform.tfvars file and just run the `terraform apply` again or will that break my whole cluster?

**Comment by eplightning** [comment]
It's not something I tested but yes, you can just change server_type in tfvars and it should work.

Keep in mind changing server_type will cause all servers to restart. If you don't want any downtime you need to do it manually.

**Comment by TheLoneDeveloper** [comment]
Alright thanks for the quick reply, I will try it

**Comment by TheLoneDeveloper** [comment]
So after manually scaling up all workers the cluster no longer works getting errors about coredns, nameservers limit, calico not ready yet and so forth.

I then tried restarting one master and then when it automatically changed the floating ip over to another master i could no longer connect to the cluster with kubectl it just said no route to host which is weird

Anyway nothing important was running on the cluster just some test apps

So i just removed the entire cluster to recreate it tomorrow and see if i made a mistake somewhere

**Comment by eplightning** [comment]
Yeah I'd need to look into it. Maybe Hetzner is doing something more than simply increasing resources and rebooting nodes.
